Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -356,6 +360,13 @@ open class TextStorage: NSTextStorage {
 
             processUnderlineDifferences(in: domRange, betweenOriginal: sourceStyle, andNew: targetStyle)
         case NSAttachmentAttributeName:
+            if sourceValue is LineAttachment || targetValue is LineAttachment {
```

Comment:
But the check here is not for nil but to see if we are handling an LineAttachment instead of an TextAttachment (images).